gravitee-apim-console-webui/src/management/api/proxy/general/response-templates/response-template-type.html

```diff
@@ -22,13 +22,13 @@ <h2>Template</h2>
   </div>
 
   <div class="gv-form-content">
-    <!-- Content-Type -->
+    <!-- Accept type -->
     <div flex-xs flex="50">
       <md-input-container class="md-block">
-        <label>Content-type</label>
+        <label>Request Accept header type</label>
         <input ng-model="$ctrl.template.type" type="text" required />
       </md-input-container>
-      <div class="hint">The content-type for which this template response must be send.</div>
+      <div class="hint">The request accept header type for which this template response must be send.</div>
```

Comment:
```suggestion
      <div class="hint">The request accept header type for which this template response should be sent.</div>
```
IMO 'should' sounds better than 'must' here and the 'send' should have a 't' instead.

Reply:
Thank's